ui/lib/kubernetes/addon/routes.js

```diff
@@ -1,5 +1,17 @@
 import buildRoutes from 'ember-engines/routes';
 
 export default buildRoutes(function () {
-  // Define your engine's route map here
+  this.route('overview');
+  this.route('roles', function () {
+    this.route('create');
+    this.route('role', { path: '/:name' }, function () {
```

Comment:
I like that idea in case a user tries to explicitly visit that url. I'll add the route with the redirect and also update the routing RFC to mention this.